src/Bicep.Core/TypeSystem/Az/AzResourceTypeProvider.cs

```diff
@@ -35,6 +35,130 @@ private static (ResourceTypeReference, Func<ResourceType>) Get_Microsoft_Resourc
                     null)));
         }
 
+        private static TypeSymbol CreateEnumType(params string[] values)
+            => UnionType.Create(values.Select(x => new StringLiteralType(x)));
+
+        private static (ResourceTypeReference, Func<ResourceType>) Get_Microsoft_Resources_deploymentScripts_2020_10_01()
+        {
+            // hand crafted from https://github.com/Azure/azure-rest-api-specs/blob/405df4e/specification/resources/resource-manager/Microsoft.Resources/stable/2020-10-01/deploymentScripts.json
+            var reference = ResourceTypeReference.Parse("Microsoft.Resources/deploymentScripts@2020-10-01");
```

Comment:
I've hard-coded this to give us some ability to play around with types before pulling in all the types!